src/Microsoft.AspNet.Identity.EntityFramework/IdentityUser.cs

```diff
@@ -87,5 +88,20 @@ public IdentityUser(string userName) : this()
         ///     Used to record failures for the purposes of lockout
         /// </summary>
         public virtual int AccessFailedCount { get; set; }
+
```

Comment:
Thanks @ajcvickers. The setter on collection nav props actually looks very weird to me. @suhasj can we remove it?